docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,28 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The <code>Portal</code> component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
```

Comment:
```suggestion
<p class="description">The portal component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
```

This doesn't work with Google:



```html
<meta name="description" content="The &lt;code&gt;Portal&lt;/code&gt; component lets you render its children into a DOM node that exists outside of its own DOM hierarchy."/>
```

Reply:
It's possible to duplicate the description, having one specific for SEO, but personally, I would encourage we thrown when `<code>` or any HTML element is used in the description.

Reply:
Sounds good. Two descriptions could get messy pretty quickly. I think we should still capitalize it here at least, since we're talking about our specific component rather than the general concept.